test/functional/feature_rbf.py

```diff
@@ -624,6 +627,24 @@ def test_no_inherited_signaling(self):
         assert_equal(True, self.nodes[0].getmempoolentry(optin_parent_tx['txid'])['bip125-replaceable'])
         assert_raises_rpc_error(-26, 'txn-mempool-conflict', self.nodes[0].sendrawtransaction, replacement_child_tx["hex"], 0)
 
+    def test_replacement_relay_fee(self):
+        outpoint = make_utxo(self.nodes[0], int(1.1*COIN))
+
+        initial_transaction = CTransaction()
+        initial_transaction.vin = [CTxIn(outpoint, nSequence=0)]
+        initial_transaction.vout = [CTxOut(1 * COIN, DUMMY_P2WPKH_SCRIPT)]
+        initial_transaction_hex = txToHex(initial_transaction)
+        self.nodes[0].sendrawtransaction(initial_transaction_hex, 0)
+
+        # Higher fee, higher fee per KB, but the replacement does not provide a relay
+        # fee conforming to node's `incrementalrelayfee` policy of 1000 sat per KB.
```

Comment:
The fee and fee per KB are lower though?

Reply:
I think the `replacement_transaction` output value is `CTxOut(int(1 * COIN - 1))` so it has a higher fee/fee per KB, as everything else is constant.

You can verify with the following diff :
```
@@ -892,6 +894,8 @@ bool MemPoolAccept::PreChecks(ATMPArgs& args, Workspace& ws)
         // The replacement must pay greater fees than the transactions it
         // replaces - if we did the bandwidth used by those conflicting
         // transactions would not be paid for.
+        LogPrintf("nModifiedFees %d\n", nModifiedFees);
+        LogPrintf("nConflictingFees %d\n", nConflictingFees);
         if (nModifiedFees < nConflictingFees)
         {
             return state.Invalid(TxValidationResult::TX_MEMPOOL_POLICY, "insufficient fee",
```

Reply:
Sorry, brainfart.